go/apps/mixnet/conn.go

```diff
@@ -21,6 +21,12 @@ import (
 
 const CellBytes = 1024 // Length of a cell
 
+const (
+	msgCell   = iota
```

Comment:
Not quite what I meant. More like

const (
  msgCell byte = iota
  dirCell
  relayCell
)